modules/ROOT/pages/troubleshoot-installation-issues.adoc

```diff
@@ -0,0 +1,156 @@
+
+
+=== Solution
+
+To resolve this error, perform the following steps:
+
+. Verify the status of the pods:
++
+`kubectl get pods --all-namespaces`
++
+All the pods in all namespaces are displayed with their status and additional details.
+. If a pod displays a “Pending” status, as illustrated in the previous screenshot, obtain additional details for the pod:
++
+`kubectl describe pods <pod_name> -n service-mesh`
+//Need a text for the screenshot here.
++
+The result of the command shows that the scheduling of the node has failed.  
+. To verify that the number of pods under the capacity section is not already filled up, run the command:
++
+`kubectl describe nodes`
+. Increase capacity of the node or amount of nodes.
++
+*Tip:* If you are using Aamzon EKS, this value is calculated from the instance type. Therefore, if you increase the instance type, the amount of pods is increased.
+. Re-run the Anypoint Service Mesh installer to complete the installation.
+
+[[service-mesh-not-found]]
+== Service Mesh Not Found
+
+When you start the Anypoint Service Mesh installation, the installer fails with the “Service Mesh not found” error.
+
+=== Causes
+
+This issue can be caused due to various reasons. 
+
+=== Diagnose
+
+To identify the command that caused the error, run the following command:
++
+`$ sh -x ./service-mesh install`
+
+=== Solution
+
+To resolve this error, download the latest version of the installer and check if the problem persists.
+
+
+[[failed-webhook-calling]]
+== Installation Failed Due to Failed WebHook Calling Error
+
+When you start the Anypoint Service Mesh installation, the installer fails with the following error:
+
+[source,text,linenums]
+----
+Error: release service-mesh failed, and has been uninstalled due to atomic being set: Internal error occurred: failed calling webhook "pilot.validation.istio.io": Post https://istio-galley.service-mesh.svc:443/admitpilot?timeout=30s: no endpoints available for service "istio-galley"
+FATAL: Installing 'service-mesh' Helm Chart!!!
+----
+
+=== Diagnose
+
+This issue can be caused due to various reasons. 
+
+To diagnose the issue:
+
+* Case 1: Verify that the master nodes have access to port 443 in worker nodes. 
+* Case 2:  Verify that no orphan resources were created from a previous installation or uninstallation:
+
+At the Command prompt, type the following command:
++
+`kubectl get clusterservicebrokers,clusterrole,psp,clusterrolebinding,mutatingwebhookconfigurations,validatingwebhookconfigurations -oname | awk '/(service-mesh|istio|catalog)/'`
++
+Any existing orphan resource are displayed.
+
+=== Solution
+
+Perform the solution based on the appropriate case as described in the Diagnose section:
+
+* Case 1: Fix your network setup to allow the master node to connect to port 443. Depending on the provider, you can accomplish this task by modifying security groups. 
```

Comment:
Replace all of this with:

"If the master node does not have access to Port 443 of the worker nodes, provide access, typically by modifying security groups."

Reply:
Done! Thanks!